2-mandatory/3-stocks.js

```diff
@@ -64,7 +90,16 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+    const priceDescriptions = [];
+
+    for (let i = 0; i < stocks.length; i++) {
+        const ticker = stocks[i].toUpperCase();
+        const highestPrice = Math.max(...closingPricesForAllStocks[i]).toFixed(2);
```

Comment:
Nice use of `Math.max`

Reply:
It's nice finding things and trying things I'm unfamiliar with on Google.